### PR TITLE
dev/core#5070 rationalise pay_later on backoffice, true for new pending

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
_A brief description of the pull request. Keep technical jargon to a minimum. Hyperlink relevant discussions._

Before
----------------------------------------
_What is the old user-interface or technical-contract (as appropriate)?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

After
----------------------------------------
_What changed? What is new old user-interface or technical-contract?_
_For optimal clarity, include a concrete example such as a screenshot, GIF ([LICEcap](http://www.cockos.com/licecap/), [SilentCast](https://github.com/colinkeenan/silentcast)), or code-snippet._

Technical Details
----------------------------------------
_If the PR involves technical details/changes/considerations which would not be manifest to a casual developer skimming the above sections, please describe the details here._

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by colemanw** [comment]
@civicrm-builder retest this please

File: CRM/Contribute/Form/Contribution.php [hunk 1]
```diff
@@ -2541,4 +2530,26 @@ private function buildRecur(): void {
     $form->add('checkbox', 'is_recur', $is_recur_label, NULL);
   }
 
+  /**
+   *
+   * @return bool|null
+   * @throws \CRM_Core_Exception
+   */
+  private function isPayLater(): bool {
```
Comment (demeritcowboy) [inline CRM/Contribute/Form/Contribution.php:2538]: This function doesn't seem to get called so it still comes up incomplete transaction. It does get called for the credit card form, but pay later makes no sense there.

File: CRM/Contribute/Form/Contribution.php [hunk 2]
```diff
@@ -2541,4 +2530,26 @@ private function buildRecur(): void {
...
+      // New contribution, base it on the status, CRM-8996
+      return $submittedStatus === 'Pending';
+    }
+    if ($this->_mode) {
+      return FALSE;
+    }
+    // (This is historical but has been questioned)
+    // If contribution_status Completed changed to FALSE, CRM-15041
+    if ($submittedStatus === 'Completed') {
+      return TRUE;
```
Comment (demeritcowboy) [inline CRM/Contribute/Form/Contribution.php:2550]: Even if it did get called, does this seem right? Shouldn't it be FALSE?

**Comment by eileenmcnaughton** [comment]
OK - I sat down & looked at this properly - https://github.com/civicrm/civicrm-core/pull/30165

**Pull request closed without merging**